json/classic/src/main/java/ch/qos/logback/contrib/json/classic/JsonLayout.java

```diff
@@ -195,7 +216,16 @@ protected Map toJsonMap(ILoggingEvent event) {
         if (this.includeFormattedMessage) {
             String msg = event.getFormattedMessage();
             if (msg != null) {
-                map.put(FORMATTED_MESSAGE_ATTR_NAME, msg);
+                if(this.needJsonFormatMessage){
```

Comment:
The if-else blocks you added in this file have formatting issues. Please fix them to be:
```java
  if (...) {
      ...
  } else {
      ...
  }
```